src/locales/en.json

```diff
@@ -532,6 +532,9 @@
   "nft_claim_claimed_content_manualDelivery": "Please wait for the author to sign and send the eBook, which may take about 1-3 days. Any update will be notified via email. Thank you for your support!",
   "nft_claim_claimed_content_mixedDelivery": "Some of the book requires author to sign and send the eBook, which may take about 1-3 days. Any update will be notified via email. For auto delivered books, visit your dashboard to start reading. Thank you for your support!",
   "nft_claim_claimed_download": "Other Ways to Obtain the eBook",
+  "nft_claim_claimed_empty_collected_help": "Can't find your purchased content? Seek {help}.",
+  "nft_claim_claimed_empty_crisp_prefilled_message": "Hi, I am unable to find the e-book I recently purchased. Could you please help me locate it? Thank you!",

```

Comment:
```suggestion
  "nft_claim_claimed_empty_crisp_prefilled_message": "Hi, I am unable to find the ebook I recently purchased. Could you please help me locate it? Thank you!",

```

Reply:
[💬 Update wording for prefilled msg](https://github.com/likecoin/liker-land/pull/1787/commits/5ffdcfd275a1a294e014ab5e45b7d1bea242f191)